### ISSUE TITLE
Accessibility Issue with empty Grid

### ISSUE THREAD

**Issue by GibletJr** [IgniteUI/igniteui-angular#9318]
## Description  
Describe the issue.

 * igniteui-angular version: 11.1.7
 * browser: Microsoft Edge (Chromium)

## Steps to reproduce  

1. Define an igx-grid element and set its _emptyGridMessage_ attribute to a value like "No data available"
2. Run an accessibility tool (like FastPass - https://accessibilityinsights.io/downloads/)

## Result  
**Path**
.igx-grid__tbody

**snippet**
<div igxgridbody="" role="rowgroup" class="igx-grid__tbody">

**How to fix**
Fix the following: Required ARIA child role not present: row

## Expected result  
No accessibility warnings

## Attachments  
![Empty Grid](https://user-images.githubusercontent.com/56270402/114629095-d4788000-9c6c-11eb-91fd-ebeda033f688.jpg)

## Potential Solution
As per the attached image, adding the _role="row"_ attribute to the interior <div> and then also added the _role="gridcell"_ to the internal <span> - which contains the content for the _emptyGridMessage_ property - was one way to resolve the alert/warning.

**Comment by GibletJr** [comment]
The product that has a dependency on IgniteUI for Angular is targeting full accessibility compliance by end of June 2021.

**Comment by GibletJr** [comment]
@kdinev - confirmed that the fix is in release in 11.1.11 and resolves the issue for igx-grid; however, found that igx-hierarchical-grid application of the same resolution.

**Comment by ChronosSF** [comment]
Should be fixed for hierarchical grid and tree grid.

**Comment by GibletJr** [comment]
@ChronosSF, any ETA on fix and release?  Our hope was to have this resolved by end of June.

**Comment by ChronosSF** [comment]
@tishko0 , @kacheshmarova , can you commit for a fix until end of June or should we take the bug for a fix in hierarchical and tree grids?

**Comment by GibletJr** [comment]
@kdinev I see the issue has been closed.  Any update on when and what release will contain this fix because I checked [11.1.15](https://github.com/IgniteUI/igniteui-angular/releases/tag/11.1.15) and found the issue still persists there.

**Comment by kdinev** [comment]
@GibletJr Hmm, I made sure that @tishko0's PRs are merged before I tag the patch versions.

@tishko0 Could you please install `11.1.15` and check why the accessibility issue is still reproducible?

**Comment by GibletJr** [comment]
@kdinev / @tishko0, I just checked and found that the developer used the emptyGridTemplate instead of emptyGridMessage.  Updated the approach to use emptyGridMessage (for consistency) and it passed the FastPass check.  Thanks!

**Comment by tishko0** [comment]
@GibletJr Thank you for your confirmation. I am glad to hear that the issue has been resolved.
